packages/xo-web/src/xo-app/jobs/overview/index.js

```diff
@@ -32,6 +29,101 @@ const jobKeyToLabel = {
   genericTask: _('customJob'),
 }
 
+const COLUMNS = [
+  {
+    itemRenderer: schedule => {
+      const { id } = schedule
+
+      return (
+        <div>
+          <span>{`${schedule.name} (${id.slice(4, 8)})`}</span>
+          <Link
+            className='btn btn-sm btn-primary ml-1'
+            to={`/jobs/schedules/${id}/edit`}
+          >
+            <Icon icon='edit' />
+          </Link>
+        </div>
+      )
+    },
+    name: _('schedule'),
+    sortCriteria: 'name',
+  },
+  {
+    itemRenderer: (schedule, userData) => {
+      const job = userData.jobs[schedule.jobId]
+
+      return (
+        job !== undefined && (
+          <div>
+            <span>{`${job.name} - ${job.method} (${job.id.slice(4, 8)})`}</span>
+            <Link
+              className='btn btn-sm btn-primary ml-1'
+              to={`/jobs/${job.id}/edit`}
+            >
+              <Icon icon='edit' />
+            </Link>
```

Comment:
Individual action as well? The labels, icons and colors should make it clear enough that we're editing the schedule or the job.

Reply:
I moved it to individual action and I changed icon and color, but it still not clear enough that's why I just added a tooltip to icon.